node_modules/oae-folders/lib/api.js

```diff
@@ -30,7 +30,8 @@ var log = require('oae-logger').logger('oae-folders-api');
 var MessageBoxAPI = require('oae-messagebox');
 var MessageBoxConstants = require('oae-messagebox/lib/constants').MessageBoxConstants;
 var OaeUtil = require('oae-util/lib/util');
-var PrincipalsAPI = require('oae-principals');
+// var PrincipalsAPI = require('oae-principals');
+var PrincipalsAPI = require('oae-principals/lib/api.group');
```

Comment:
I know but it doesn't work (executing unit tests) with "var PrincipalsAPI = require('oae-principals');" that's why I changed it for "var PrincipalsAPI = require('oae-principals/lib/api.group');" ... that's weird

Reply:
Which test specifically fails? Off the top of my head, could this be caused by a naming conflict in PrincipalsAPI (if there is now methods with the same name in both api.group and api.user)?

Reply:
All unit tests related with oae-folders fails and there is no similar methods in api.group and api.user

Reply:
Hmm, that's very strange. I'll look into it... Could you paste the stacktrace for one of the failures, please?

Reply:
1) Folders - Activity
      verify the update activity:

      Uncaught AssertionError [ERR_ASSERTION]: false == true
      + expected - actual

      -false
      +true
      
    at /Users/clozan01/Documents/Projet_Delete/Hilary/node_modules/oae-folders/lib/test/util.js:355:20
    at Request._callback (/Users/clozan01/Documents/Projet_Delete/Hilary/node_modules/oae-rest/lib/util.js:233:20)
    at Request.self.callback (/Users/clozan01/Documents/Projet_Delete/Hilary/node_modules/oae-rest/node_modules/request/request.js:372:22)
    at Request.emit (events.js:159:13)
    at Request.<anonymous> (/Users/clozan01/Documents/Projet_Delete/Hilary/node_modules/oae-rest/node_modules/request/request.js:1310:14)
    at Request.emit (events.js:164:20)
    at IncomingMessage.<anonymous> (/Users/clozan01/Documents/Projet_Delete/Hilary/node_modules/oae-rest/node_modules/request/request.js:1258:12)
    at IncomingMessage.emit (events.js:164:20)
    at endReadableNT (_stream_readable.js:1062:12)
    at process._tickCallback (internal/process/next_tick.js:152:19)

Reply:
Thanks. I can't replicate this locally, but considering that the error seems to be raised from oae-rest, it could be caused by some change to that module... In any case, I think this line should probably be reverted since it's hiding the error rather than fixing it IMO.